Doc/library/ossaudiodev.rst

```diff
@@ -447,4 +447,3 @@ The remaining methods are specific to audio mixing:
    microphone input::
 
       mixer.setrecsrc (1 << ossaudiodev.SOUND_MIXER_MIC)
-
```

Comment:
Do not remove existing lines

Reply:
I can revert this, but I'm curious, why do you want two empty lines at the bottom?

Reply:
I am sorry. I didn't look at the file. If it is at bottom, you can keep it. I am worried about introducing unnecessary git diffs in the middle of a file.